### PR TITLE
CRC Forward Error Correction for 1 and 2bit bursts

### PR DESCRIPTION
**Pull request by vosk**

### CODE REVIEW

**Comment by vosk** [comment]
Sample correction:
```
[INFO] ----------------------------------------------------------------------
[INFO] Datetime: 2021-03-23 17:59:52.602834 UTC
[INFO] SNR: 0.00 dB
[INFO] Downlink Format (DF): 17 Extended Squitter
[INFO] CRC: Failed PI-CRC = 8033033
[DEBUG] FEC: detected faulty bits [48]
[DEBUG] FEC: Conservative error correction:True
[INFO] ----------------------------------------------------------------------
[INFO] Datetime: 2021-03-23 17:59:52.602834 UTC
[INFO] SNR: 0.00 dB
[INFO] Downlink Format (DF): 17 Extended Squitter
[INFO] Capability (CA): 5 Level 2 or Above Transponder, Can Set CA 7, In Air
[INFO] Address Announced (AA): 4241d8
[INFO] Callsign: VPCZA   
[INFO] Type Code (TC): 11 Airborne Position
[INFO] Surveillance Status (SS): 0 No Condition Information
[INFO] Time: 0 Not Synced to 0.2s UTC Epoch
[INFO] Latitude: 35.322441489009535 N
[INFO] Longitude: 23.517228796126997 E
[INFO] Altitude: 43000 ft
```

**Comment by mhostetter** [comment]
Thank you for the pull request! I like the concept. I will review the PR tomorrow.

File: python/decoder.py [hunk 1]
```diff
@@ -601,7 +631,7 @@ def check_parity(self):
                     self.log("info", "CRC", "Passed")
                     return 1
                 else:
-                    self.log("info", "CRC", Fore.RED + "Failed", "PI-CRC = {}".format(pi-crc))
+                    self.log("info", "CRC", Fore.RED + "Failed", "PI-CRC = {}".format(pi^crc))
```
Comment (mhostetter) [inline python/decoder.py:634]:
```suggestion
                    self.log("info", "CRC", Fore.RED + "Failed", "PI^CRC = {}".format(pi^crc))
```

File: python/decoder.py [hunk 2]
```diff
@@ -651,7 +681,7 @@ def check_parity(self):
...
```
Comment (mhostetter) [inline python/decoder.py:684]:
```suggestion
                    self.log("info", "CRC", Fore.RED + "Failed", "PI^CRC = {}".format(pi^crc))
```

**Review by mhostetter — COMMENTED** [review]
I like the PR. I just made a small print suggestion. Add those tweaks and I'll merge it in. I appreciate it.

**Comment by vosk** [comment]
Done :)

**Pull request merged**